src/stacks-hierarchy/navigation-types.ts

```diff
@@ -112,6 +112,13 @@ type Root_PurchasePaymentWithCreditCardScreenParams = PaymentParams & {
   paymentMethod: CardPaymentMethod;
 };
 
+type Root_ConfirmationScreenParams = {
+  message: string;
+  // Time that must be wait until onComplete is called (in milliseconds)
+  delayBeforeCompleted?: number;
+  nextScreen: NextScreenParams<'Root_TabNavigatorStack'>;
```

Comment:
Do you need the generics here anymore with the updated typing for `NextScreenParams`?